dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/dataset/DatasetService.java

```diff
@@ -226,11 +221,6 @@ private AuthenticatedUser retrieveAuthenticatedUser() {
     }
 
     private Dataset updateDatasetEmbargoDate(Dataset dataset, Date embargoDate) throws IllegalStateException {
-        if(dataset.isLocked()) {
-            logger.log(Level.WARNING, "Dataset is locked. Cannot perform update embargo date");
-            throw new IllegalStateException(getDatasetLockedMessage(dataset));
-        }
```

Comment:
This will allow for change in embargo when any lock is active. As far as I remember we wanted to allow update of embargo only in case of `InReview` lock. For other type of locks we should still throw an exception (similar to what we do in `UpdateDatasetVersionCommand`).

Reply:
Ok, yeah, I think you're right. Changed it to only byPass `InReview` locks.